### DISCUSSION TITLE
How to improve carbon components SASS compile speed

### DISCUSSION THREAD

**Discussion by nSedrickm** [carbon-design-system/carbon#8526] [Help] [answered]
Hello, Greetings.

I have been using carbon-components-react to build a [dashboard](https://github.com/SMSwithoutBorders/SMSwithoutBorders_Admin) Please I have found the sass files take a really long time to compile. Please what can I do to improve the compile speed and reduce resource consumption?

**Comment by tw15egan** [comment, marked as answer]
Hey there @nSedrickm, I recommend taking a look at this step in the Carbon Tutorial that has some recommendations to help build up your style compile time

https://www.carbondesignsystem.com/developing/react-tutorial/step-1#import-carbon-component-styles

Hope that helps!

**Comment by jnm2377** [comment]
@nSedrickm definitely second what TJ said above, but also wanted to share that we just released our v11 beta, which uses sass modules for much faster compile times! 😄

**Comment by nSedrickm** [comment]
Thank you
